packages/stepper/src/Stepper.story.tsx

```diff
@@ -13,21 +13,24 @@ storiesOf('Stepper', module)
     <div style={{ width: 1000 }}>
       <Stepper
         currentStep={number('Step', 0, { min: 0, max: 6 })}
+        completedStepsShown={number('Completed steps shown', 2, { min: 1 })}
         maxDisplayedSteps={5}
       >
-        <Step>Overview</Step>
-        <Step>Configuration</Step>
-        <Step>Update</Step>
-        <Step>Install</Step>
-        <Step>Billing</Step>
-        <Step>Address</Step>
-        <Step>Confirmation</Step>
+        <div>Overview</div>
+        <div>Configuration</div>
+        <div>Update</div>
+        <div>Install</div>
+        <div>Billing</div>
+        <div>Address</div>
+        <div>Confirmation</div>
```

Comment:
One major-ish change I made is to not require the use of the `<Step>` component, because below is the implementation of the old `<Step>` component:

```
export const Step = function Step({
  children,
}: {
  children: StepContents;
}): StepElement {
  return <>{children}</>;
};
```

I didn't really see the point in that, so I changed it to just wrap each child in the new Step logic.